Dockerfile

```diff
@@ -29,8 +29,13 @@ RUN curl --fail --location --remote-name https://github.com/pmem/ndctl/archive/v
 RUN tar zxvf v${NDCTL_VERSION}.tar.gz && mv ndctl-${NDCTL_VERSION} ndctl
 WORKDIR /ndctl
 RUN ./autogen.sh
-RUN ./configure ${NDCTL_CONFIGFLAGS}
-RUN make install
+# We install into /usr/local (keeps content separate from OS) but
+# then symlink the .pc files to ensure that they are found without
+# having to set PKG_CONFIG_PATH. The .pc filed doesn't container an -rpath
```

Comment:
> The .pc filed doesn't container an -rpath

some thing(s) not right in that sentence.

Reply:
"container" - a popular word, it sneaks into sentences even when it doesn't belong there ;-}

Fixed.